### DISCUSSION TITLE
Receive "Please select proof type 'Other' for this image." while the page doesn't have the choice.

### DISCUSSION THREAD

**Discussion by 2018liuzhiyuan** [community/community#68728] [GitHub Education]
After I uploaded my photo, the page didn't have a choice to select the type. 
![image](https://github.com/community/community/assets/50035957/e847c966-26fb-4cef-8d19-900eed082e68)

**Comment by swei2001** [comment]
Me too.

**Reply by Mr-lonely0** [reply]
Me too. Have you solved this problem?

**Reply by swei2001** [reply]
Yes, I wrote my information down and resubmit and it worked.

**Reply by Mr-lonely0** [reply]
If you mean school information in github profile, I updated it but still not work

**Comment by moayad-star** [comment]
If you are having trouble with the image format, you can use this site [image.online-convert](https://image.online-convert.com/) Or other similar sites

**Reply by swei2001** [reply]
Sorry, what do you mean? Change the format of the image from jpg to others?

**Reply by moayad-star** [reply]
Written in the message
> we accept **_JPEGs_**
---
If your/him image format is different from the required format, please try this solution

**Reply by Mr-lonely0** [reply]
I upload the jpeg photo which is required, but still meet the same problem. There seems no choices to select "Other" type in this page:
![QQ截图20231016153912](https://github.com/community/community/assets/67233215/2b369c5e-658e-4feb-91fa-d22ccf946dc2)

**Comment by yuanshanmian** [comment]
I succeeded! I think maybe this website might have some questions, which is why the 'proof type' button isn't visible. My guess is that the default proof type is 'student ID,' so if you upload any other material, it could cause this error. To resolve this, you can use your Student ID Card（学生证） with the effective date clearly visible, and make sure it's in English. Save the file as 'Student ID Card.jpg,' and upload. You should be all set.

**Reply by z1xy2** [reply]
Your advice works, Thanks for sharing!

**Reply by yuanshanmian** [reply]
haha, and then my friends find that if only changing the name of pic to ''student's ID card'' works too😂

**Reply by Tim4Dec** [reply]
Hello, could you please tell me which browser you are using to open this page? My page doesn't even have the 'upload an image' option, so I can't rename the picture :(

**Reply by yuanshanmian** [reply]
I use Chrome, do not use VPN, and your location need to be in your campus

**Reply by Tim4Dec** [reply]
Thank you for your reply. I am also using the latest version of Chrome, but I still cannot access the interface with an upload option :( . Could you please tell me what operating system you are using?

**Reply by yuanshanmian** [reply]
hmm, I use win11,  But I think maybe it's blame on the Internet. Because I remember I have encountered the same problem that did not show the bottom too. Maybe you can change the browser software like Edge, and different network connection mode

**Reply by yizeyi18** [reply]
Is the proof type buttom on the page once the page loads, or only appears after uploading the image? I got overfused finding the buttom.
